Source/NSMenu.m

```diff
@@ -476,9 +476,8 @@ - (void) _setGeometry
   if (_menu.horizontal == YES)
     {
       NSRect screenFrame = [[NSScreen mainScreen] frame];
-      origin = NSMakePoint (32, screenFrame.size.height
-                            - [_aWindow frame].size.height);
-      origin.y += screenFrame.origin.y;
+      origin = NSMakePoint (35, screenFrame.size.height
```

Comment:
My preference would be to extract the magic number `35` into a constant names something like `APPLICATION_MENU_LEFT_OFFSET` or `GLOBAL_MENU_WIDTH` or something like that.

Reply:
Re: key equivalents: The "quit" and "log out" items in GWorkspace's file menu are leaving, but for other things where shift must be held, the NSShiftKeyMask has to be added to the keyEquivalentModifierMask property of the MenuItem -- it now treats capital and lower case letters the same, as in the Mac.

Re: Commented out code: I figured leave it there for a while in case I needed it quickly.

Re: Magic number: That makes sense, I'll push that now.